lua/astrocommunity/lsp/coc-nvim/init.lua

```diff
@@ -2,6 +2,11 @@ return {
   {
     "neoclide/coc.nvim",
     branch = "release",
+    init = function()
+      vim.g.coc_global_extensions = {
+        "coc-marketplace",
```

Comment:
This should be configured in AstroCore `options.g` table which exposes it to the user rather than a closed box `init` function that cannot be extended.